core/common/src/main/java/alluxio/util/ConfigurationUtils.java

```diff
@@ -247,4 +271,175 @@ public static boolean masterHostConfigured() {
   public static String valueAsString(String value) {
     return value == null ? "(no value set)" : value;
   }
+
+  /**
+   * Returns an instance of {@link AlluxioConfiguration} with the defaults and values from
+   * alluxio-site properties.
+   *
+   * @return The default set of Alluxio properties loaded from the site-properties file
+   */
+  public static AlluxioProperties defaults() {
+
+    if (sDefaultProperties == null) {
+      synchronized (DEFAULT_PROPERTIES_LOCK) { // We don't want multiple threads to reload
+        // properties at the same time.
+
+        // Check if properties are still null so we don't reload a second time.
+        if (sDefaultProperties == null) {
+          reloadProperties();
+        }
+      }
+    }
+    return sDefaultProperties.copy();
+  }
+
+  /**
+   * Forces the static site properties to be reloaded from disk.
```

Comment:
Can "Forces reload" be shortened to just "Reloads"?

Isn't the point of reloading the site properties that they are *not* static? Otherwise what's the point in reloading them?